projects/demo/src/pages/stackblitz/files/styles.css

```
@@ -1,7 +1,43 @@
-/* TODO: make better basic customization */
+* {
+    --tui-text-01: rgba(27, 31, 59, 1);
+    --tui-text-03: rgba(27, 31, 59, 0.4);
+
+    --tui-radius-m: 0.75rem;
+
+    --tui-base-01: #fff;
+    --tui-base-03: #ededed;
+    --tui-primary: #526ed3;
+}
+
 input {
```

Could we use the same customization for textarea too ?

```suggestion
input, textarea {
```

See https://tinkoff.github.io/maskito/recipes/textarea#latin